src/main/java/org/carlmontrobotics/Constants.java

```diff
@@ -19,27 +25,69 @@ public static final class Drivetrain {
 	}
 	
 	public static final class Arm {
+
+	
+
+		//Motor port
+		public static final int ARM_MOTOR_PORT_MASTER = 13;
+		public final static int ARM_MOTOR_PORT_FOLLOWER = 18;
+		//Config for motors
+		public static final boolean MOTOR_INVERTED_MASTER = true; //Todo: find all these (they are definetely wrong)
+		public static final boolean MOTOR_INVERTED_FOLLOWER = false;
+		public static final double ROTATION_TO_RAD = 2 * Math.PI;
+		public static final boolean ENCODER_INVERTED = false;
 		
-		//all angles in rot here
-		public static final double intakeAngle = -.1;
-		public static final double ampAngle = .3;
-		public static final double speakerAngle = .4;
-		//if needed
-//		public static final trapAngle = 80;
 		
-		//Motor Controllers: pid, FF
-		public static final double[] pidVals = new double[] { 0.1, 0.0, 0.1 };
-		// Feedforward
+		public static final int MAX_VOLTAGE = 12;
+		public static final double ENCODER_OFFSET_RAD = 0;
+		
+		//TODO: finish understand why this is broken public static final Measure<Angle> INTAKE_ANGLE = Degrees.to(-1);
+
+		// USE RADIANS FOR THE ARM
+		public static final double INTAKE_ANGLE = Units.degreesToRadians(0);
+		public static final double AMP_ANGLE = Units.degreesToRadians(105);
+		public static final double SUBWOFFER_ANGLE = Units.degreesToRadians(24);
+		public static final double SAFE_ZONE_ANGLE = Units.degreesToRadians(24);
+		public static final double PODIUM_ANGLE = Units.degreesToRadians(24);
+		public static final double CLIMBER_UP_ANGLE = Units.degreesToRadians(24);
+		public static final double CLIMBER_DOWN_ANGLE = Units.degreesToRadians(24);
+
+
+		//PID, Feedforward, Trapezoid
+		public static final double kP = 0.1;
+		public static final double kI = 0.1;
+		public static final double kD = 0.1;
 		public static final double kS = 0.1;
 		public static final double kG = 0.1;
 		public static final double kV = 0.1;
 		public static final double kA = 0.1;
-		
+		public static final double IZONE = 4;
 		public static final double MAX_FF_VEL = 1; // rot / s
-		public static final double MAX_FF_ACCEL = 1; // rot / s^2
-		public static TrapezoidProfile.Constraints trapConstraints = new TrapezoidProfile.Constraints(MAX_FF_VEL, MAX_FF_ACCEL);
+		public static final double MAX_FF_ACCEL = 1; // rot / s^2 
+
+		//if needed
+		public static final double COM_ARM_LENGTH_METERS = 0.381 ;
+		public static final double ARM_MASS_KG = 9.59302503;
+		
+		
+		public static TrapezoidProfile.Constraints TRAP_CONSTRAINTS = new TrapezoidProfile.Constraints(MAX_FF_VEL, MAX_FF_ACCEL);
+		//other0;
+
+		public static final double MARGIN_OF_ERROR = Math.PI/18;
+		
+		//Boundaries
+		public static final double ARM_TELEOP_MAX_GOAL_DIFF_FROM_CURRENT_RAD = 0; //placeholder
+		public static final double POS_TOLERANCE_RAD = 0; //placeholder
+		public static final double VEL_TOLERANCE_RAD_P_SEC = 0; //placeholder
```

Comment:
I suggest you set the `POS_TOLERANCE_RAD` to 1 count of an encoder. The way an encoder works is that there are a ton of "counts" in the encoder. The encoder cannot read values between these counts. Say for example an encoder had 360 counts, then each count = 1 degree. It cannot read values between 0 and 1 degree, 1 and 2 degrees, and so on. You will need to do some conversion and search up how many counts the encoder has. As always, if the tolerance is too low and makes the arm shake then make the tolerance higher. For `VEL_TOLERANCE_RAD_P_SEC` you can divide by the time it takes for a robot loop to run.